scripts/docker/clean-logs.sh

```diff
@@ -35,6 +35,9 @@ while true; do
     -type f -mtime +"${RETENTION}" -name '*.log' -print0 | \
     xargs -0 rm -f
 
+  find "${DIRECTORY}"/logs \
+    -type d -mtime +"${RETENTION}" -empty -delete
```

Comment:
I think you should not use `-mtime` here at all. `-mtime` for directory is updated when file is removed for example - with means that if a file in the folder is removed, then it will remain empty fore RETENTION days. That makes very little sense. I think it makes much more sense if simply all empty directories are removed. There is no point in keepning them threre.

Reply:
You are right.